mintlify/connector
# Login page

# Summary
- Implements the login view UI
- Adds login to log in and save over all locally saved documentation into an account

<img width="1552" alt="Screen Shot 2022-07-10 at 12 51 10 PM" src="https://user-images.githubusercontent.com/44352119/178154152-80aacd49-ec6f-40be-a5bf-1653d4ba0fb0.png">

### Important logic to note
- When signing in with existing local docs: two things can happen 1) if there is already previous account with that email, all docs remain in that local account and user logs into the existing account or 2) if there is no previous account, all local docs are ported over to the new account

### Testing
- Make sure that `ISDEV = false`
- Try logging in with an account where there are already docs saved locally that isn't already part of an organization
- Try logging in with an account where there are already docs saved locally where the user already is part of another organization
- Logout works
